.travis.yml

```diff
@@ -15,6 +16,55 @@ env:
   - TEST_SUITE=brakeman
 matrix:
   fast_finish: true
+  include:
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.5.8
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=vmdb PARALLEL=true
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.5.8
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=brakeman
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.6.6
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=vmdb PARALLEL=true
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.6.6
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=brakeman
```

Comment:
By the same logic in https://github.com/ManageIQ/manageiq/pull/20812/files#r568699782, I think we can drop the brakeman tests.  They shouldn't be different based on architecture, so any security issues would be found in the x86-64 runs, and the results here would just be redundant.